components/items/ItemsGrid/ItemsGridImageTokenEntity.vue

```diff
@@ -200,9 +200,8 @@ const onClickShoppingCart = async () => {
 const onClickListingCart = async () => {
   const nftsToProcess = await getTokensNfts([props.entity])
 
-  for (const nft of nftsToProcess) {
```

Comment:
This part make a weird loop of the same items on some generative nfts

Reply:
cc @Jarsen136 ?

Reply:
thanks for the clarification, i've added missing variables